### PR TITLE
Update device-control-removable-storage-access-control.md

### PR DESCRIPTION
**Pull request by tewchen**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 1]
```diff
@@ -61,11 +61,11 @@ For each device property, see **Device Properties** section above for more detai
     - Primary ID
         - RemovableMediaDevices
         - CdRomDevices
-    - DeviceId
-    - HardwareId
-    - InstancePathId
-    - FriendlyNameId
-    - SerialNumberId
+    - DeviceId: add wildcard support (* or ?) anywhere in the text value in the version 4.18.2105 or later
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md:64]:
```suggestion
    - DeviceId: add wildcard support (* or ?) anywhere in the text value in version 4.18.2105 or later
```

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 2]
```diff
@@ -61,11 +61,11 @@ For each device property, see **Device Properties** section above for more detai
...
+    - HardwareId: add wildcard support (* or ?) anywhere in the text value in the version 4.18.2105 or later
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md:65]:
```suggestion
    - HardwareId: Add wildcard support (* or ?) anywhere in the text value in version 4.18.2105 or later
```

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 3]
```diff
@@ -61,11 +61,11 @@ For each device property, see **Device Properties** section above for more detai
...
+    - InstancePathId: add wildcard support (* or ?) anywhere in the text value in the version 4.18.2105 or later
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md:66]:
```suggestion
    - InstancePathId: Add wildcard support (* or ?) anywhere in the text value in version 4.18.2105 or later
```

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 4]
```diff
@@ -61,11 +61,11 @@ For each device property, see **Device Properties** section above for more detai
...
+    - FriendlyNameId: add wildcard support (* or ?) anywhere in the text value in the version 4.18.2105 or later
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md:67]:
```suggestion
    - FriendlyNameId: Add wildcard support (* or ?) anywhere in the text value in version 4.18.2105 or later
```

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 5]
```diff
@@ -61,11 +61,11 @@ For each device property, see **Device Properties** section above for more detai
...
+    - SerialNumberId: add wildcard support (* or ?) anywhere in the text value in the version 4.18.2105 or later
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md:68]:
```suggestion
    - SerialNumberId: Add wildcard support (* or ?) anywhere in the text value in version 4.18.2105 or later
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @tewchen, could you please commit my suggested changes? Many thanks.

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md (hunk 5 above)
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md]:
```suggestion
    - DeviceId: Add wildcard support (* or ?) anywhere in the text value in version 4.18.2105 or later
    - HardwareId: Add wildcard support (* or ?) anywhere in the text value in version 4.18.2105 or later
    - InstancePathId: Add wildcard support (* or ?) anywhere in the text value in version 4.18.2105 or later
    - FriendlyNameId: Add wildcard support (* or ?) anywhere in the text value in version 4.18.2105 or later
    - SerialNumberId: Add wildcard support (* or ?) anywhere in the text value in version 4.18.2105 or later
```

**Comment by yogkumgit** [comment]
@tewchen Please implement the suggested feedback. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@v-smandalika Please merge this PR. Thanks

**Comment by v-smandalika** [comment]
@yogkumgit @AshaIyengar21 @claydetels19 


Hi Yogesh Kumar,

Based on the chain of the notification emails, we understand that we need to carry out the changes through a new ticket as we do not have the privileges to merge PRs and that too one that is worked on by an SME/POC.

I shall take this up with my lead and project manager and shall revert. Hope this plan is okay with you.


Thanks,
Siddarth

**Comment by yogkumgit** [comment]
@v-smandalika OK. please either merge or close the PRs associated to you.

**Comment by v-smandalika** [comment]
Hi @yogkumgit,

Apologies that we could not take this up with our Principal Manager Daniel Simpson.

Just FYI, we from the documentation team do not have the privileges to merge PRs; hence, we have taken this up with Daniel Simpson.  We will be getting an idea as to whether new tickets are to be created for us to import changes from this PR into our new PRs.

We have notified Daniel Simpson about this PR's status, and we will be reverting to you tomorrow.


Thanks,
Siddarth

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Comment by chrisda** [comment]
Updated information already in topic in the [Prepare your endpoints](https://docs.microsoft.com/microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control#prepare-your-endpoints) section:

Deploy Removable Storage Access Control on Windows 10 devices that have antimalware client version **4.18.2103.3 or later**.

- **4.18.2104 or later**: Add SerialNumberId, VID_PID, filepath-based GPO support, ComputerSid
- **4.18.2105 or later**: Add Wildcard support for HardwareId/DeviceId/InstancePathId/FriendlyNameId/SerialNumberId, the combination of specific user on specific machine, removeable SSD (a SanDisk Extreme SSD)/USB Attached SCSI (UAS) support

Closing

**Pull request closed without merging**